includes/RemoteWiki.php

```diff
@@ -59,6 +62,7 @@ public function __construct( string $wiki ) {
 		$this->dbcluster = $wikiRow->wiki_dbcluster;
 		$this->category = $wikiRow->wiki_category;
 		$this->experimental = $wikiRow->wiki_experimental;
+		$this->hookRunner = new CreateWikiHookRunner( MediaWikiServices::getInstance()->getHookContainer() );
```

Comment:
Same thing here (above `$this->config`)?

Reply:
Done. Thanks!